underlay/src/main/java/bio/terra/tanagra/filterbuilder/impl/core/utils/GroupByCountSchemaUtils.java

```diff
@@ -89,17 +89,13 @@ public static Map<Entity, List<Attribute>> getGroupByAttributesPerOccurrenceEnti
 
   public static BinaryOperator toBinaryOperator(
       DTUnhintedValue.UnhintedValue.ComparisonOperator comparisonOperator) {
-    switch (comparisonOperator) {
-      case COMPARISON_OPERATOR_EQUAL:
-        return BinaryOperator.EQUALS;
-      case COMPARISON_OPERATOR_LESS_THAN_EQUAL:
-        return BinaryOperator.LESS_THAN_OR_EQUAL;
-      case COMPARISON_OPERATOR_GREATER_THAN_EQUAL:
-        return BinaryOperator.GREATER_THAN_OR_EQUAL;
-      case COMPARISON_OPERATOR_BETWEEN:
```

Comment:
I think it's useful to detail all the cases, even if they have the same effect.

Reply:
added it as another case with similar exception